caffe2/operators/quantized/int8_sigmoid_op.h

```diff
@@ -0,0 +1,256 @@
+#ifndef CAFFE2_INT8_SIGMOID_OP_H_
+#define CAFFE2_INT8_SIGMOID_OP_H_
+#include "caffe2/core/context.h"
+#include "caffe2/core/operator.h"
+#include "caffe2/core/tensor_int8.h"
+#include "caffe2/operators/quantized/int8_simd.h"
+#include "caffe2/operators/quantized/int8_utils.h"
+
+namespace caffe2 {
+namespace int8 {
+namespace {
+
+/*
+ * Implementation based on TensorFlow Lite kernels:
+ * - Repo: https://github.com/tensorflow/tensorflow
+ * - Path: tensorflow/contrib/lite/kernels/internal/optimized/optimized_ops.h
+ * - Hash: d4ad9c73969c45d1a224ebfc43eb645b9860216b
+ */
+
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+void QuantizeMultiplierGreaterThanOne(
+    double double_multiplier,
+    int32_t* quantized_multiplier,
+    int* left_shift) {
+  CHECK(double_multiplier > 1.);
+  const double q = std::frexp(double_multiplier, left_shift);
+  auto q_fixed = static_cast<int64_t>(Round(q * (1ll << 31)));
+  CHECK(q_fixed <= (1ll << 31));
+  if (q_fixed == (1ll << 31)) {
+    q_fixed /= 2;
+    ++*left_shift;
+  }
+  CHECK_GE(*left_shift, 0);
+  CHECK_LE(q_fixed, std::numeric_limits<int32_t>::max());
+  *quantized_multiplier = static_cast<int32_t>(q_fixed);
+}
+
+inline int32_t MultiplyByQuantizedMultiplierGreaterThanOne(
+    int32_t x,
+    int32_t quantized_multiplier,
+    int left_shift) {
+  using gemmlowp::SaturatingRoundingDoublingHighMul;
+  return SaturatingRoundingDoublingHighMul(
+      x * (1 << left_shift), quantized_multiplier);
+}
+
+int CalculateInputRadius(int input_integer_bits, int input_left_shift) {
```

Comment:
I think I've seen these functions (`QuantizeMultiplierGreaterThanOne`, `MultiplyByQuantizedMultiplierGreaterThanOne`, `CalculateInputRadius`) in several files. Please move them to `int8_utils.h` to avoid duplication